pdns/recursordist/docs/settings.rst

```diff
@@ -884,6 +884,45 @@ To prevent this, add a Negative Trust Anchor (NTA) for this zone in the `lua-con
 If this forwarded zone is signed, instead of adding NTA, add the DS record to the `lua-config-file`_.
 See the :doc:`dnssec` information.
 
+.. _setting-forward-zones-catalogs
+
+``forward-zones-catalogs``
+--------------------------
+.. versionadded:: 4.10.0
+-  List of 'zone=IP[:port]' pairs, separated by commas
+
+The contents of the specified ``catalog zones`` will be retrieved from
+the servers at the specified IPs (or IP:port combinations).  Each
+entry in the catalog zones will be added to the `forward-zones`_ list.
+
+.. code-block:: none
+
+    forward-zones-catalogs=catalog1.example.org=203.0.113.210
+    forward-zones-catalogs+=catalog2.example.org=[2001:DB8::1:3]:5300
+
+The catalog zones themselves will be added to the `allow-notify-for`_
+list, but when NOTIFY queries are received for them the recursor will
+not perform any cache-clearing operations; instead it will retrieve
+the SOA record for the catalog zone in the NOTIFY query, and if the
+serial number is higher than the serial number last seen, the zone
+contents will be retrieved. Any zones in the `forward-zones`_ list
+which were learned from the previous version of the catalog zone, and
+which are no longer present in the catalog zone, will be removed from
+the `forward-zones`_ list; any zones which have been added to the
+catalog zone will be added to the `forward-zones`_ list.
+
+Forwarded queries for zones obtained from a catalog zone will be sent
```

Comment:
Could we allow for specifying a different server/ip to forward to?

Reply:
It's certainly possible, now that the recursor supports config in YAML, more complex configurations are practical to express.

Reply:
I'm not sure if recursor (consumer) will be able to read catalog zone from another consumer (secondary auth), but this would the clear use case.
Usually in complex architectures, producer of catalog usually not gets the queries, consumer receive the user traffic.

Having the option to forward to different set of IPs, would be great.